src/hazardous/aead/chacha20poly1305.rs

```diff
@@ -686,7 +686,7 @@ mod private {
 				// The usize that padding() returns should always
 				// be what remains to make input a multiple of 16 in length.
 				fn prop_padding_result(input: Vec<u8>) -> bool {
```

Comment:
NIT: Now that padding takes `usize` we can have quickcheck generate those instead of `input: Vec<u8>`: `input: usize`.